app/src/main/java/com/sopt/peekabookaos/presentation/pickModify/PickModifyActivity.kt

```diff
@@ -0,0 +1,42 @@
+package com.sopt.peekabookaos.presentation.pickModify
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import com.sopt.peekabookaos.R
+import com.sopt.peekabookaos.databinding.ActivityPickModifyBinding
+import com.sopt.peekabookaos.util.binding.BindingActivity
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class PickModifyActivity :
+    BindingActivity<ActivityPickModifyBinding>(R.layout.activity_pick_modify) {
+    private lateinit var itemDeco: PickModifyDecoration
+    private lateinit var pickShelfAdapter: PickModifyAdapter
+    private val viewModel by viewModels<PickModifyViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding.vm = viewModel
+        initAdapter()
+        initItemDecoration()
+    }
+
+    private fun initAdapter() {
+        pickShelfAdapter = PickModifyAdapter { position, item ->
+            viewModel.updateSelectedItemState(item)
+            viewModel.selectState.value?.let {
+                pickShelfAdapter.updateSelectedPosition(
+                    position,
+                    it
+                )
```

Comment:
```suggestion
                pickShelfAdapter.updateSelectedPosition(position, it)
```
이렇게 하면 더 가시적으로 보일 것 같습니다.